### PR TITLE
Update integration test CentOS to 7.3

### PR DESCRIPTION
**Pull request by branden**
## High Level Description

See #1305.

## Related Issues

  - [DCOS-13207](https://jira.mesosphere.com/browse/DCOS-13207) Test on CentOS 7.3

## Checklist for all PR's

  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here: This is a change to the OS used by some integration tests.
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by branden** [comment]
@mesosphere-mergebot bump-ee

File: test_util/aws.py [hunk 1]
```diff
@@ -500,15 +500,15 @@ def get_host_ips(self):
                   'us-east-1': 'ami-fa9b9390',
                   'us-west-1': 'ami-12b3ce72',
                   'us-west-2': 'ami-edf11b8d'},
```
Comment (branden) [inline test_util/aws.py]: @lloesche Do we have AMIs for CentOS 7.3 without DC/OS prereqs?

**Comment by branden** [comment]
@mesosphere-mergebot bump-ee

File: test_util/aws.py (hunk 1 above)
Comment (lloesche) [inline test_util/aws.py]: We don't. The 7.3 AMI with prereqs is based on a 7.2 base AMI where we run `yum -y --releasever=7.3.1611 update`

**Comment by lloesche** [comment]
This could be closed in favour of #1329

**Comment by branden** [comment]
👍

**Pull request closed without merging**